src/Agent.Worker/TaskRunner.cs

```diff
@@ -525,5 +538,24 @@ private void PrintTaskMetaData(Definition taskDefinition)
             ExecutionContext.Output($"Help         : {taskDefinition.Data.HelpUrl ?? taskDefinition.Data.HelpMarkDown}");
             ExecutionContext.Output("==============================================================================");
         }
+
+        private bool IsInputContainsSecret(string inputValue)
+        {
+            string maskedString = HostContext.SecretMasker.MaskSecrets(inputValue);
+            return maskedString.Contains("***");
+        }
+
+        private bool ValidateInjectedTaskInputs(Dictionary<string, string> inputs)
```

Comment:
Could you please make the name more specific? Basically it simply checks if inputs contain any secrets - although validation is something more generic